string agent_id
RobotConfiguration robot_configuration
---
bool success
